source/common/network/listen_socket_impl.h

```diff
@@ -26,9 +26,9 @@ class SocketImpl : public virtual Socket {
       fd_ = -1;
     }
   }
-  void addOption(OptionPtr&& option) override {
+  void addOption(OptionConstSharedPtr&& option) override {
```

Comment:
As a shared_ptr, this probably shouldn't be an rvalue.

Reply:
Good catch, this was global search/replace snafu.

Reply:
You probably already saw but this issue is in a bunch of other places.